docs/en/enterprise-edition/policy-reference/azure-policies/azure-networking-policies/bc-azure-239.adoc

```diff
@@ -0,0 +1,62 @@
+== Azure Machine learning workspace is not configured with private endpoint
+
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+| a8e6f787-f937-42eb-9f71-ffc4cb17ba8a
+
+|Checkov ID
+| https://github.com/bridgecrewio/checkov/blob/main/checkov/arm/checks/resource/AzureMLWorkspacePrivateEndpoint.py[CKV_AZURE_239]
+
+|Severity
+|MEDIUM
+
+|Subtype
+|Build
+
+|Frameworks
+|ARM,Bicep
+
+|===
+
+=== Description
+
+This policy is checking to confirm that an Azure Machine learning workspace is not set up with a private endpoint. Using a private endpoint can pose a risk as it exposes the system to access via the private network only, hence limiting the accessibility, scalability, and flexibility of the system. Private endpoints further risk a single point of failure, which can directly affect the availability of the whole application or service.
```

Comment:
@tsmithv11 : **The description seems to defeat the purpose of the title, making private endpoints an issue rather than a positive**. Also the fix below configures endpoints. Having said that, here is the revised description.

This policy detects whether an Azure Machine Learning workspace is not configured with a private endpoint. While private endpoints offer security benefits, they can also limit accessibility, scalability, and flexibility. Additionally, they potentially risk a single point of failure, which can directly affect the availability of the entire application or service.

Reply:
Good catch! I took yours and updated it to reflect that this is checking for the presence of a private endpoint not the lack of it.